packages/shared/src/modules/user-follows/libs/types/user-follows-follow-parameters-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type UserFollowsFollowParametersDto = {
```

Comment:
+ do not forget to change file name
```suggestion
type UserFollowsParamsDto = {
```

Reply:
I can keep UserFollowsParametersDto if needed

Reply:
@D1monB yes, let's move on with UserFollowsParametersDto 👍